root/static/scripts/common/components/EditorLink.js

```diff
@@ -6,10 +6,23 @@
 const {trim} = require('lodash');
 const React = require('react');
 
+const {l} = require('../i18n');
 const entityHref = require('../utility/entityHref');
 const isolateText = require('../utility/isolateText');
 
+const MissingEditorLink = () => {
+  return (
+    <span className="deleted tooltip" title={l('This editor is missing from this server, and cannot be displayed correctly.')}>
+      {isolateText(l('[missing editor]'))}
+    </span>
+  );
+};
+
 const EditorLink = ({editor, content, avatarSize, subPath}) => {
+  if (!editor || editor.id == 1 && editor.name == '[missing editor]') {
```

Comment:
I think the `!editor` check is sufficient if you remove the fake editor and let it stay `null`.

Reply:
Done in https://github.com/metabrainz/musicbrainz-server/pull/792/commits/da3428846427526719bf706eaa3af9c2bc6afefe